### PR TITLE
Create ml toggle

### LINKED ISSUE #87: Create a machine learning toggle
Create a toggle that allows users to switch between the regular chat and ml chat. This should switch the route that the frontend uses to send messages.

### PR DESCRIPTION
**Pull request by joshcwinton**
Create a button that changes whether messages get sent to standard chatbot or ml chatbot.

Probably should replace with something that is more obviously checked or unchecked.

Issue: Closes #87 

### Frontend:
- Add button component
- Function that toggles state attribute ml
- Based on the value of this.state.ml edit axios call

### CODE REVIEW

File: frontend/src/components/Chat.jsx [hunk 1]
```diff
@@ -124,6 +128,19 @@ class Chat extends Component {
     });
   };
 
+  setChecked = () => {
+    console.log(this.state.ml);
```
Comment (JasperCheung) [inline frontend/src/components/Chat.jsx:132]: make this more descriptive? Console.log("ml")/Console.log("chatbot")

**Review by JasperCheung — APPROVED** [review]
Looks good.

File: frontend/src/components/Chat.jsx (hunk 1 above)
Comment (joshcwinton) [reply frontend/src/components/Chat.jsx:132]: meant to remove this line 😅

Comment (JasperCheung) [reply frontend/src/components/Chat.jsx:132]: that works too

**Pull request merged**